chain/jsonrpc-primitives/src/types/chunks.rs

```diff
@@ -6,7 +6,7 @@ use serde_json::Value;
 pub enum ChunkReference {
     BlockShardId {
         block_id: near_primitives::types::BlockId,
-        shard_id: near_primitives::types::ShardId,
+        shard_id: near_primitives::types::ShardOrd,
```

Comment:
Just for the reference, I would apply the same reasoning about `shard_id: ShardId` I expressed above. Thus, it looks fine to me now.